woocommerce/woocommerce-ios
# Show Variations screen on tapping "Add price to your variations..." warning row

Closes #4849

## Description
“Add price to your variations to make them visible…” warning in `ProductFormViewController` was not tappable and an issue was reported to make it tappable and also launch the Variations screen on tap. 
This change includes the code to make the warning row actionable and also launch the `ProductVariationsViewController` on tap.

## Testing
You should add a Product with a variant, which doesn't have a price set. (Please refer to the attached recording.)

1. Go to the `Products` tab.
2. Tap on the product which has a variant and no price set.
3. In the tableview, you should see the "Add price to your variations to make them visible on your store" warning, just below the `Variations` row.
4. Tap on "Add price to your variations to make them visible on your store" warning and observe that `Variations` screen is shown.
5. Navigating back and tapping on `Variations` row should also end up showing the `Variations` screen.


https://user-images.githubusercontent.com/524475/135368677-cc101da8-4c29-43dc-9270-3ba702a9d2c2.mov

Update release notes:
